common/hooks/pre-pkg/04-generate-runtime-deps.sh

```diff
@@ -44,6 +44,7 @@ store_pkgdestdir_rundeps() {
                 printf -- "${_curdep} " >> ${PKGDESTDIR}/rdeps
             done
         fi
+        cat ${PKGDESTDIR}/rdeps | xargs -n1 | sort | xargs > ${PKGDESTDIR}/rdeps
```

Comment:
Reading and writing same file in one pipeline looks like race which can truncate file, did you test?

It can be simplified to 

        printf -- "${_curdep} "
      done | sort > ${PKGDESTDIR}/rdeps
    fi

without truncating file inside `if`.

Reply:
good catch, it worked fine in my testing, but I think it may cause issues on huge packages. I've updated it. How's that look?